common/check_version.sh

```diff
@@ -24,7 +32,7 @@ go_on_after_warning () {
 
     echo "Do you want to continue? The resulting package is not likely to work correctly."
     echo "It is recommended that you use older version of packaging scripts instead"
-    echo "(older version is available at https://github.com/leo-project/leofs_package/tree/1.0.0)"
+    echo "(older version is available at https://github.com/leo-project/leofs_package/tree/1.2.0)"
```

Comment:
https://github.com/leo-project/leofs_package/releases/tag/1.2.0 seems more suitable URL than the above as download links are there.